src/ccid_usb.c

```diff
@@ -1550,7 +1550,7 @@ static void *Multi_PollingProc(void *p_ext)
 							change = (slot_status & 2) ? "status changed" : "no change";
 
 							DEBUG_COMM3("slot %d status: %d",
-								s + b*4, slot_status);
+								s + slot, slot_status);
```

Comment:
By the way, is the calculation ("s + b*4") correct here? It's a bit confusing that the variable called "slot" was calculated as just "b*4".